app/javascript/src/NewService/components/NewServiceForm.jsx

```diff
@@ -29,22 +28,19 @@ const NewServiceForm = (props: Props) => {
     setFormMode(event.currentTarget.value)
 
   const formToRender = () => formMode === 'manual'
-    ? <ServiceManualForm template={template} formActionPath={adminServicesPath} apiap={apiap} backendApis={backendApis} />
-    : <ServiceDiscoveryForm formActionPath={providerAdminServiceDiscoveryServicesPath} apiap={apiap} setLoadingProjects={setLoadingProjects} />
-
-  const title = apiap ? 'New Product' : 'New API'
+    ? <ServiceManualForm template={template} formActionPath={adminServicesPath} backendApis={backendApis} />
+    : <ServiceDiscoveryForm formActionPath={providerAdminServiceDiscoveryServicesPath} setLoadingProjects={setLoadingProjects} />
 
   return (
     <React.Fragment>
-      <h1>{title}</h1>
+      <h1>New Product</h1>
```

Comment:
Right, we are not using I18N here, it was just for "Product" or "Service", no?

Reply:
Correct